plugins/design-system/pages/DesignSystemPage.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+import "../styles/shame.less";
+
+const SDK = require("../SDK").getSDK();
+
+const { Page } = SDK.get(["Page"]);
+
+class DesignSystemPage extends Component {
+  render() {
+    return <Page>{this.props.children}</Page>;
+  }
+}
```

Comment:
I'm fine with keeping it for now. Maybe there was a good reason, but I can't think of any right now except that it prevents us from having to import the SDK in every design system page, maybe.